docker-scripts/update-phoenix.sh

```diff
@@ -46,4 +46,4 @@ echo "✅ Phoenix Wallet has been updated."
 
 echo "🚀 Launching the node..."
 
-java -classpath /app/burst.jar brs.Burst
+java -classpath /app/signum-node.jar brs.Burst
```

Comment:
Consider changing to `java -jar /app/signum-node.jar` in the future, so if the main class is changed it will still work.